src/app/commands/scoap-squad/ScoapSquad.ts

```diff
@@ -0,0 +1,131 @@
+import {
+	ApplicationCommandPermissionType,
+	CommandContext,
+	CommandOptionType,
+	SlashCommand,
+	SlashCreator,
+} from 'slash-create';
+import roleIds from '../../service/constants/roleIds';
+import ServiceUtils from '../../utils/ServiceUtils';
+import CreateNewScoapPoll from '../../service/scoap-squad/CreateNewScoapPoll';
+import ValidationError from '../../errors/ValidationError';
+import discordServerIds from '../../service/constants/discordServerIds';
+
+module.exports = class ScoapSquad extends SlashCommand {
```

Comment:
I've seen that the `export` keyword is also used in this project. Is there a reason for using CommonJS here?

Reply:
This is something taken from the slash-create example https://slash-create.js.org/#/docs/main/latest/examples/command. I think it should be fine keeping it as module.exports but I'll leave it to @tikisailor to decide

Reply:
yep, just went with the existing codebase on this.